_data/simple-icons.json

```
@@ -661,6 +661,11 @@
             "hex": "9146FF",
             "source": "https://luna.amazon.com"
         },
+        {
+            "title": "Amazon Music",
+            "hex": "25D1DA",
+            "source": "https://music.amazon.com"
```

```suggestion
            "source": "https://music.amazon.com",
            "guidelines": "https://artists.amazonmusic.com/tools"
```
We should add the guidelines entry